tutorials/hyperspectral/Plot-Hyperspectral-Spectra.Rmd

```diff
@@ -148,11 +132,13 @@ by a factor of 10, 100, 10000, etc. This `scale factor` will be noted in the dat
 
 ```{r pull-scale-factor }
 
-# grab scale factor
-scaleFact <- reflInfo$`Scale Factor`
+# grab scale factor from the Reflectance attributes
+reflectanceAttr<- h5readAttributes(f,"/SJER/Reflectance/Reflectance_Data" )
+
+scaleFact = reflectanceAttr$Scale_Factor
```

Comment:
Check throughout for <- assignment operator

Reply:
Yes! Double-checked and changed.